packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,69 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+## PatternFly 5.4
+The final minor release within PatternFly 5 is now live&mdash;here are the highlights. 
+
+Next up is PatternFly 6! Don't forget to keep an eye on [our staging site](https://staging-v6.patternfly.org/) for updates. 
+
+### Promoted package versions
+
+### Fixed bugs
+
+We addressed some remaining bugs that you helped us catch. 
+
+- Added support for additional label info on horizontal forms.
+- Added a missing border on compound expansion table rows.
+- Fixed `<ExpandableSection>` errors that were caused by `variant="truncate"`.
+- Updated `paginationAriaLabel` to stop adding "items" to the end of labels.
+- Fixed unexpected wrapping in the pagination component. 
+- Added `shouldFocusFirstMenuItemOnOpen` to the dropdown component, to fix autofocus issues.
+- Fixed spacing issues with progress stepper's help text title.
+
+### New severity icons
+
+We created new severity icons, which can be used to communicate the severity of an error or issue with a data source that is linked to a UI element. These icons have been added to both our [HTML](https://www.npmjs.com/package/@patternfly/icons) and [React](https://www.npmjs.com/package/@patternfly/react-icons) packages. They are also available for design use in our [PatternFly 6 Figma design kit.](https://www.figma.com/community/file/1357060119827689328/patternfly-6-components) 
```

Comment:
I'm not sure what this package is - https://www.npmjs.com/package/@patternfly/icons. It looks like it just has JS import files for each icon though. The HTML icon set technically isn't a separate package, it just comes with core. So you could update the link to point to core maybe? (https://www.npmjs.com/package/@patternfly/patternfly) I'm curious what others think - cc @nicolethoen

Reply:
ahh tbf I somehow navigated to that url myself just by guessing 😂 I think it makes sense to just point to core instead